packages/headless/src/features/commerce/facets/facet-set/interfaces/request.ts

```diff
@@ -59,7 +59,7 @@ export type LocationFacetRequest = BaseCommerceFacetRequest<
 
 export type BaseCommerceFacetRequest<Value, Type extends FacetType> = Pick<
   FacetRequest,
-  'facetId' | 'field' | 'numberOfValues' | 'isFieldExpanded'
```

Comment:
We pick `numberOfValues` here, but then override it a few lines below. As we actually want the `numberOfValues` to be optional, we keep the `numberOfValues` definition below.